ga4gh/datamodel/reads.py

```diff
@@ -223,21 +277,12 @@ class SimulatedReadGroup(AbstractReadGroup):
     """
     def __init__(self, parentContainer, localId, randomSeed, numAlignments=2):
         super(SimulatedReadGroup, self).__init__(parentContainer, localId)
-        self._numAlignments = numAlignments
+        self._parentContainer = parentContainer
 
     def getReadAlignments(self, referenceId=None, start=None, end=None):
-        for i in range(self._numAlignments):
+        for i in range(self._parentContainer.getNumAlignedReads()):
```

Comment:
I think we should keep this as the per-readgroup numAlignments, not inheriting it from the parent.